Source/Scene/ModelExperimental/ModelExperimentalRuntimePrimitive.js

```diff
@@ -184,6 +186,10 @@ ModelExperimentalRuntimePrimitive.prototype.configurePipeline = function (
   const pointCloudShading = model.pointCloudShading;
   const hasAttenuation =
     defined(pointCloudShading) && pointCloudShading.attenuation;
+  const hasPointCloudStyle =
+    primitive.primitiveType === PrimitiveType.POINTS &&
+    (defined(style) || hasAttenuation);
```

Comment:
Actually, to amend the second bullet: there can be a style that affects the glTF model, but it can't reference any built-in properties (`POSITION`, etc.) the way that the style of a `pnts` tileset can